src/sentry/api/endpoints/organization_events.py

```diff
@@ -91,6 +94,29 @@ def handle_results(self, request, organization, project_ids, results):
 
 
 class OrganizationEventsV2Endpoint(OrganizationEventsEndpointBase):
+    def build_cursor_link(self, request, name, cursor):
+        # The base API function only uses the last query parameter, but this endpoint
+        # needs all the parameters, particularly for the "field" query param.
+        querystring = u"&".join(
+            u"{0}={1}".format(urlquote(query[0]), urlquote(value))
+            for query in request.GET.lists()
+            if query[0] != "cursor"
+            for value in query[1]
+        )
```

Comment:
You should be able to use `six.moves.urllib.parse.urlencode(params, doseq=True)` to encode `request.GET` if you copy the dictionary and remove cursor first.

Reply:
Is that better?

Reply:
It would use fewer lines of code, but this works too :smile: